src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/security/ResourceServerConfiguration.kt

```diff
@@ -31,6 +35,7 @@ class ResourceServerConfiguration {
           "/swagger-resources/configuration/ui",
           "/swagger-resources/configuration/security",
         ).forEach { authorize(it, permitAll) }
+        authorize(anyRequest, hasAuthority(authorisedRole))
         authorize(anyRequest, authenticated)
```

Comment:
Do we need this line?: `authorize(anyRequest, authenticated)`

Reply:
I think not. If I remove it the tests still pass and also in theory someone cannot have an authority without being authenticated. I will remove it